client/src/main/java/tds/config/TimeLimits.java

```diff
@@ -0,0 +1,157 @@
+package tds.config;
+
+/**
+ * Represents a record from the {@code session.timelimits} view.
+ * <p>
+ *     While the data for instances of this class is pulled from the {@code session} database, it resides in the
+ *     Config service because almost all of the fields for the {@code session.timelimits} view actually come from the
+ *     {@code configs.client_timelimits} table.  The only fields that come from the {@code session._externs} table are
+ *     the client name and enviroment.  Both of these fields are also stored in the {@code configs.client_timelimits}
+ *     table, although they store different values:
+ *
+ *     SELECT clientname, _efk_testid, environment FROM configs.client_timelimits;
+ *
+ *     # clientname, _efk_testid, environment
+ *       'SBAC_PT', NULL, NULL
+ *       'SBAC', NULL, NULL
+ *
+ *     SELECT clientname, _efk_testid, environment FROM session.timelimits;
+ *
+ *     # clientname, _efk_testid, environment
+ *       'SBAC', NULL, 'Development'
+ *       'SBAC_PT', NULL, 'Development'
+ *
+ *     For reasons that are not entirely clear, The legacy code specifically references the {@code session._externs}
+ *     table for the client name and environment fields, so that behavior has been carried forward.
+ * </p>
+ */
+public class TimeLimits {
+    private String clientName;
+    private String environment;
+    private String assessmentId;
+    private Integer examRestartWindowMinutes;
+    private Integer examDelayDays;
+    private Integer interfaceTimeoutMinutes;
+    private Integer requestInterfaceTimeoutMinutes;
+    private Integer taCheckinTimeMinutes;
+
```

Comment:
Seems like this object would be immutable.  Is that safe to say?  If so, I think we talked about using the builder pattern to remove the setters and not have a constructor with 8 parameters.